docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -621,6 +621,15 @@ Set to `true` to add the Content-Security-Policy header to your requests. CSP al
 
 Set Content Security Policy template used when adding the Content-Security-Policy header to your requests. `$NONCE` in the template includes a random nonce.
 
+### content_security_policy_report_only
+
+Set to `true` to add the Content-Security-Policy-Report-Only header to your requests. CSP in Report Only mode allows to experiment with policies by monitoring their effects without enforcing.
+Both policies can be enabled simultaneously.
+
+### content_security_policy_template
+
+Set Content Security Policy Report Only template used when adding the Content-Security-Policy-Report-Only header to your requests. `$NONCE` in the template includes a random nonce.
```

Comment:
I'm not sure what you are trying to say here. Is it: "Set the Content Security Policy Report Only template that is used when you add the Content-Security-Policy-Report-Only header to your requests."?

Reply:
@chri2547 I updated the wording of this phrase, hopefully it makes more sense now.

Reply:
👍